src/modules/fees-collector/services/fees-collector.transaction.service.ts

```diff
@@ -72,4 +79,50 @@ export class FeesCollectorTransactionService {
             .buildTransaction()
             .toPlainObject();
     }
+
```

Comment:
this one it does not do what it says, if remove is true then it removes the contract not adding it

Reply:
maybe a better name will be handleKnownContract and also for the mutation

Reply:
or have 2 different mutiations

Reply:
done